src/EditorFeatures/Core/Implementation/Suggestions/FixMultipleOccurrencesService.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions
+{
+    /// <summary>
+    /// Service to compute and apply <see cref="FixMultipleCodeAction"/> code fixes.
+    /// </summary>
+    [ExportWorkspaceServiceFactory(typeof(IFixMultipleOccurrencesService), ServiceLayer.Host), Shared]
```

Comment:
is this code fix service that doesnt require view? would it be possible for us to have only 1 code fix service?